### DISCUSSION TITLE
File not updating on DreamHost

### DISCUSSION THREAD

**Discussion by FCWolf** [iterate-ch/cyberduck#15696] [Q&A] [answered]
Using a Mac I am attempting to update a file in the DreamHost file manager. Dragging the new file to the Cyberduck window changes the time stamp in the file manager but the file itself is not changed from the previous version. (There is no problem when uploading the file directly to DreamHost.) Any thoughts appreciated.

**Comment by tatze96** [comment]
Does the upload fails silently without displaying an error message?
Does this behavior occur in combination with specific files (file size, file extension)?

**Reply by FCWolf** [reply, marked as answer]
Thanks for your reply. This was user error. I did not notice the drop-down allowing me to select the “overwrite” option. Sorry to not have updated my comment sooner.
